### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by elizaveta-lomteva**

### CODE REVIEW

File: dataflow/flex-templates/salesforce-to-bigquery/README.md [hunk 1]
```diff
@@ -337,9 +344,11 @@ The template allows the user to supply the following optional parameters:
 - `secretStoreUrl` - URL to Salesforce credentials in HashiCorp Vault secret storage in the format
   'http(s)://vaultip:vaultport/path/to/credentials'.
 - `vaultToken` - Token to access HashiCorp Vault secret storage.
-- `expansionService` - Python expansion service in format host:port, needed for RunInference transforms. You can leave it empty if all of your extra packages are available in pip.
-  You can create Compute Engine VM using pre-build [expansion service image](https://hub.docker.com/layers/akvelon/dna-accelerator/expansion-service/images/sha256-045986791106f035993819d3ff3b66ac182489a45c14eba78c6f5077ff11910f?context=explore),
-  and configure port 8088 open for incoming connections in GCP Firewall. Please see [Expansion Service setup](https://github.com/akvelon/DnA_accelerators/blob/main/dataflow/ml/salesforce/pytorch/anomaly_detection/README.md#expansion-service) documentation for more details on how to build your expansion service.
+- `expansionService` - Python expansion service in format host:port, needed for RunInference transforms.
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/flex-templates/salesforce-to-bigquery/README.md]: deleted

File: dataflow/flex-templates/salesforce-to-bigquery/README.md [hunk 2]
```diff
@@ -337,9 +344,11 @@ The template allows the user to supply the following optional parameters:
...
+  > **_NOTE:_** You can leave it empty if all of your extra packages are available in pip and you're passing them in `withExtraPackages` method.
+  
+  You can create Compute Engine VM using pre-build [Expansion Service image](https://hub.docker.com/layers/akvelon/dna-accelerator/expansion-service-2.47/images/sha256-a7a0869a6a9ee9ef7036a37c408234aa923fb14acd07f95ed180bbe757141138?context=explore),
+  and configure port 8088 open for incoming connections in GCP Firewall. Please see [Expansion Service setup documentation](https://github.com/akvelon/DnA_accelerators/blob/main/dataflow/ml/salesforce/pytorch/anomaly_detection/README.md#expansion-service) for more details on how to build your expansion service.
 
 You can provide the next secured parameters directly instead of providing HashiCorp Vault parameters:
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/flex-templates/salesforce-to-bigquery/README.md]: Updated the list of template's parameters by adding the Salesforce security parameters section with the ability to choose how to provide them: directly or through HashiCorp.
Please take a look

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md [hunk 3]
```diff
@@ -19,35 +18,59 @@ instead of training the model from the ground.
...
 
-For the demo we use [mock salesforce data](../../../../data/salesforce) with client opportunities.
-To demonstrate the pipeline, we encode the values with PyTorch then apply a clustering model to detect the anomalies.
+For the demo, we use [mock Salesforce data](../../../../data/salesforce) with client opportunities.
+To demonstrate the pipeline, we encode the values with PyTorch and then apply a clustering model to detect the anomalies.
+
+## How to use the Anomaly Detection transform in the pipeline
+To use a custom transform in the multi-language pipeline you can use two options:
+<details>
+  <summary>Pass published to pip Transforms to the pipeline</summary>
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: Replaced with "Use PythonExternalTransform to stage PyPI package dependencies" confirming the statement in the [documentation](https://beam.apache.org/documentation/programming-guide/#1312-creating-cross-language-python-transforms)

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md [hunk 4]
```diff
@@ -19,35 +18,59 @@ instead of training the model from the ground.
...
+  </br>
+  
+  If your custom Python transform is published in pip, you can use the `withExtraPackages` method of the `RunInference` class and specify the dependency in the arguments.
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: Updated the section

File: dataflow/ml/salesforce/tfx/regression/README.md [hunk 5]
```diff
@@ -6,28 +6,42 @@ Run `train.ipynb` for training
...
-In this pipeline we use Tensorflow and TFX to build a deep learning model for predicting the opportunity window in days
+In this pipeline, we use Tensorflow and TFX to build a deep-learning model for predicting the opportunity window in days
 
 Prepared models and parameters tailored for our data are located in `./pretrained`
 
-### Expansion service
+### Custom Python Transforms usage
+To use Custom Python Transform you have two options:
+<details>
+  <summary>Passing published to pip Transforms to the pipeline</summary>
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/tfx/regression/README.md]: resolved

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md [hunk 6]
```diff
@@ -19,35 +18,59 @@ instead of training the model from the ground.
 
 ## Data
 
-For the demo we use [mock salesforce data](../../../../data/salesforce) with client opportunities.
-To demonstrate the pipeline, we encode the values with PyTorch then apply a clustering model to detect the anomalies.
+For the demo, we use [mock Salesforce data](../../../../data/salesforce) with client opportunities.
+To demonstrate the pipeline, we encode the values with PyTorch and then apply a clustering model to detect the anomalies.
+
+## How to use the Anomaly Detection transform in the pipeline
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: In the section below we only talk about how to use anomaly detection in Java multi-language pipeline.
Updated the documentation, please take a look

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md [hunk 7]
```diff
@@ -63,12 +86,12 @@ docker run -p 8088:8088 exp_service
 gcloud builds submit ./pipeline --tag gs://{gc_project_name}/{project_name}/{tag}:latest
 ```
 We use [Google Cloud CLI](https://cloud.google.com/sdk/docs/install) for building the image. After that the image
-will be available by the following path, and we can use it for launching Cloud Compute instance.
+will be available by the following path, and we can use it for launching the Cloud Compute instance.
 
 ### 3. Start a cloud compute instance:
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: Resolved

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md [hunk 8]
```diff
@@ -19,35 +18,59 @@ instead of training the model from the ground.
...
+  <summary>Use Custom Expansion Service</summary>
 
 ## Expansion service
 
 ### [Beam documentation on expansion service](https://beam.apache.org/documentation/glossary/#expansion-service)
 
-In order to make the inference on python available in multilang pipeline, all model inference and data preprocessing is 
-packed into a custom PTransform that is hosted on expansion service.
+In order to make the inference on Python available in a multi-language pipeline, all model inference and data preprocessing is 
+packed into a custom PTransform that is hosted on an expansion service.
```
[the remark this answers is not in the document]
Comment (elizaveta-lomteva) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: I agree with the comment, replaced "hosted with "installed". More precisely.

File: dataflow/ml/salesforce/pytorch/anomaly_detection/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (AKosolapov) [reply dataflow/ml/salesforce/pytorch/anomaly_detection/README.md]: Thanks!

**Pull request merged**